### DISCUSSION TITLE
Station - Cargo customer view

### DISCUSSION THREAD

**Discussion by freddepettersson** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#40] [Ideas]
Hi
I would like to see a better overveiw when looking at the new cargo view in station mode
![image](https://user-images.githubusercontent.com/80942768/114317428-5ac78180-9b08-11eb-900d-2e9caa8b987c.png)

My wish are to see in a singleview
- Cargo customers , track , in and out going cargo incl volyme units  
- active year maybe are not needed in the overview page

**Comment by fjallemark** [comment]
I planned to make a report that show all relevant information of a station; tracks, customers and cargo flow. However, i like your idea and will look into it to see how it looks.
